### DISCUSSION TITLE
Adding handy to web-frameworks benchmark

### DISCUSSION THREAD

**Discussion by cyrusmsk** [andrewlalis/handy-httpd#20] [Show and tell]
Hi @andrewlalis 

I found one quite popular repo with simple web frameworks benchmarks  https://github.com/the-benchmarker/web-frameworks

Currently there is only vibe.d example for D language.
So I’ve prepared some examples with other frameworks in D.
Are you fine with adding your solution there?
Because I really don’t have any experience in web and http things: I would appreciate any suggestions from you to improve code performance or any other advice.
You can find code here: https://github.com/cyrusmsk/httpbench_examples/tree/main/handy_example

thanks in advance

**Comment by andrewlalis** [comment]
Yes, I'm fine with adding it; I wasn't able to find any specification for what exactly the test setup endpoints should look like, but what you have here should be pretty much as fast as my framework will get.
